.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: ruby
-rvm:
-  - 2.4.5
```

Comment:
Can you set the `.ruby-version` file to this version of ruby to keep testing against the same version?

Reply:
Oh, we want to keep it at 2.4.5? If so, we need to nail down the dependency on `activesupport` then, as it will need 2.5+ (and right now there is no max dependency requirement there).

Reply:
Oh wait I see better what’s going on here. We should test across multiple versions of ruby and active support here. Not just completely remove the `rvm`. Could you build out the ruby/active support matrix that’s necessary for travis?

Reply:
I haven't forgotten about this.. 

I understand how to get multiple ruby versions tested, and assume would follow [this](https://www.rubydoc.info/github/thoughtbot/appraisal#travis-ci-integration) for active support versions (i.e. use Appraisal) but will likely need help on how to get the matrix to omit older ruby versions for newer active support versions? 

Also before further work here, would like to touch base with you on what the vision is for open source repos in regards to what gets run where.. since Jenkins is running the tests across the 3 appraisal versions today (and ideally without much duplication).

Reply:
Here is a PR off of yours that sets this up with the full matrix of multiple ruby versions and appraisal. https://github.com/Invoca/contextual_logger/pull/33

Reply:
Thanks. So for open source where we are running on travis, do we still want to also run the N stages of tests on Jenkins? Is jenkins still going to be the source of truth for adding the `clean-build` tag?

Reply:
No, I don't think Jenkins should be used at all because it's not public and won't ever be public.  TravisCI should be the source of truth for green builds of our public gems/projects imo.